app/views/todos/_todo_modal.html.haml

```diff
@@ -0,0 +1,20 @@
+#todomodal.modal.fade{"aria-hidden" => "true", "aria-labelledby" => "exampleModalLabel", :role => "dialog", :tabindex => "-1"}
+  .modal-dialog{:role => "document"}
+    .modal-content
+      .modal-header
+        %h5#exampleModalLabel.modal-title New ToDo
+        %button.close{"aria-label" => "Close", "data-dismiss" => "modal", :type => "button"}
+          %span{"aria-hidden" => "true"} ×
+      .modal-body
+        = form_with model: @todo do |form|
+          %div
+            = form.label :title
+            %br/
+            = form.text_field :title
+          %div
+            = form.label :text
+            %br/
+            = form.text_area :text
+          %div
+            = form.submit
+            
```

Comment:
Don't forget to leave an empty line at the end. Additionally, you may want to enable this flag in the VS Code settings https://stackoverflow.com/a/30884298/4983910

Reply:
If you take a closer look, the last line has some spaces, please enable the above flag in the VS Code so you don't have to deal with deleting trailing spaces manually.